packages/app/documents/__tests__/__fixtures__/workflow-with-two-initial-nodes.json

```diff
@@ -0,0 +1,122 @@
+{
+    "roles": ["Author", "Post-Publish Reviewer"],
+    "nodes": [
+        {
+            "privileges": [
+                {
+                    "privilege": ["create"],
+                    "role": "Author"
+                }
+            ],
+            "id": "Init",
+            "readyForUse": false
+        },
+        {
+            "privileges": [
+                {
+                    "privilege": ["edit", "comment"],
+                    "role": "Author"
+                }
+            ],
+            "id": "Draft",
+            "readyForUse": false
+        },
+        {
```

Comment:
This is the initial state we'll be testing with, "Another Init Node", in addition to "Draft"

Reply:
Nice :)